scripts/duplicate_schema_id.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+
+function extractSchemaIdsFromFile(filePath) {
+	const content = fs.readFileSync(filePath, 'utf-8');
+	const schemaIdRegex = /\$id:\s*['"]([^'"\s,]+)['"]/g;
+	let match;
+	const ids = [];
+	while ((match = schemaIdRegex.exec(content)) !== null) {
+		const line = content.substring(0, match.index).split('\n').length;
+		ids.push({ id: match[1], file: filePath, line: line });
+	}
+	return ids;
+}
+
+function checkDuplicateIds(directoryPath) {
```

Comment:
I thought that the convention we use in the codebase is to use `const` instead of `function`?

I personally find `function` to be cleaner, but then it is inconsistent with the rest of the codebase.